### PR TITLE
[style] fix lengths overflow in format_message lib

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/libs/format_message.coffee [hunk 1]
```diff
@@ -94,17 +93,32 @@ exports.formatContent = (message) ->
...
         rich = text.replace urls, '<a href="$1" target="_blank">$1</a>'
 
         # Tranform Separation chars into HTML
-        rich = rich.replace /^>>>>>[^>]?.*$/gim, '<span class="quote5">$&</span><br />\r\n'
-        rich = rich.replace /^>>>>[^>]?.*$/gim, '<span class="quote4">$&</span><br />\r\n'
-        rich = rich.replace /^>>>[^>]?.*$/gim, '<span class="quote3">$&</span><br />\r\n'
-        rich = rich.replace /^>>[^>]?.*$/gim, '<span class="quote2">$&</span><br />\r\n'
-        rich = rich.replace /^>[^>]?.*$/gim, '<span class="quote1">$&</span><br />\r\n'
+        for n in [5..1]
+            rex = new RegExp "^#{Array(n+1).join('>')}[^>]?.*$", 'gim'
```
Comment (nono) [inline client/app/libs/format_message.coffee]: Too bad that `String.prototype.repeat` is not available on all the browsers

**Comment by nono** [comment]
@m4dz do you want to keep this PR? Or can we close it?

**Comment by m4dz** [comment]
Let's close it, we'll see when we go to work on emails again :)

**Comment by m4dz** [comment]
Let's close it, we'll see when we go to work on emails again :)

**Pull request closed without merging**